contrib/builder/deb/amd64/generate.sh

```diff
@@ -116,6 +117,24 @@ for version in "${versions[@]}"; do
 
 	echo >> "$version/Dockerfile"
 
+	if [ -n "$INSTALL_SECCOMP" ]; then
+		cat >> "$version/Dockerfile" <<-'EOF'
+			ENV SECCOMP_VERSION 2.3.0
+			RUN set -x \
+			    && export SECCOMP_PATH="$(mktemp -d)" \
+			    && curl -fsSL "https://github.com/seccomp/libseccomp/releases/download/v${SECCOMP_VERSION}/libseccomp-${SECCOMP_VERSION}.tar.gz" \
+			          | tar -xzC "$SECCOMP_PATH" --strip-components=1 \
+			    && ( \
+			         cd "$SECCOMP_PATH" \
+			         && ./configure --prefix=/usr/local \
+			         && make && make install && ldconfig \
+			        ) \
+			    && rm -rf "$SECCOMP_PATH"
+		EOF
+       fi
```

Comment:
Seccomp is required to build runc, and the version we need isn't available in a lot of packaging systems so we need to build on our own.